plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterPriceSlider.php

```diff
@@ -143,15 +138,13 @@ class="max"
 						<input
 							class="max"
 							type="text"
-							data-wp-bind--value="woocommerce/product-filter-price::state.formattedMaxPrice"
+							data-wp-bind--value="state.formattedMaxPrice"
 							data-wp-on--focus="actions.selectInputContent"
-							data-wp-on--input="actions.debounceSetPrice"
-							data-wp-on--change--set-price="woocommerce/product-filter-price::actions.setMaxPrice"
-							data-wp-on--change--navigate="woocommerce/product-filters::actions.navigate"
+							data-wp-on--input="actions.debounceSetMaxPrice"
 							aria-label="<?php esc_attr_e( 'Filter products by maximum price', 'woocommerce' ); ?>"
 						/>
 					<?php else : ?>
-					<span data-wp-text="woocommerce/product-filter-price::state.formattedMaxPrice"><?php echo wp_kses_post( $formatted_max_price ); ?></span>
+					<span data-wp-text="state.formattedMaxPrice"><?php echo wp_kses_post( $formatted_max_price ); ?></span>
```

Comment:
The Interactivity API handles server-side rendering but you need to populate the state in the server.

```php
wp_interactivity_state( '...', array(
    'maxPrice'          => $max_price,
    'formattedMaxPrice' => wc_price( $max_price, array( 'decimals' => 0 ) ),
));
```

```suggestion
					<span data-wp-text="state.formattedMaxPrice"></span>
```

Reply:
Thanks, I applied this in 6b9fedd68041